checklists/avd_checklist.en.json

```diff
@@ -518,64 +302,496 @@
+            "guid": "5784b6ca-5e9e-4bcf-8b54-c95459ea7369",
+            "severity": "Low",
+            "link": "https://learn.microsoft.com/en-us/azure/storage/files/storage-files-smb-multichannel-performance"
+        },
+        {
+            "category": "Storage",
+            "subcategory": "Azure NetApp Files",
+            "text": "If NetApp Files storage is required, check storage service availability in your specific region.",
+            "description": "If a second region is required for DR purposes verify NetApp availability in there as well.",
+            "guid": "4a359836-ee79-4d6c-9d3a-364a5b7abae3",
+            "severity": "Medium",
+            "link": "https://azure.microsoft.com/global-infrastructure/services/"
+        },
+        {
+            "category": "Storage",
+            "subcategory": "Azure NetApp Files",
+            "text": "If Azure NetApp Files storage is used, check Active Directory Site name setting in the Active Directory Connection configuration",
+            "description": "An Active Directory Site should be created for the Azure virtual network environment where Azure NetApp Files (ANF) subnet will be created, and that site name should be specified in the ANF connection property when executing the join procedure as explained in the reference article.",
+            "guid": "6647e977-db49-48a8-bc35-743f17499d42",
+            "severity": "High",
+            "link": "https://docs.microsoft.com/en-us/azure/azure-netapp-files/create-active-directory-connections"
         },
         {
             "category": "Monitoring and Management",
             "subcategory": "Management",
-            "text": "Assess the requirement for host pool auto-scaling capability",
-            "description": "The scaling tool provides a low-cost automation option for customers who want to optimize their session host VM costs. You can use the scaling tool to schedule VMs to start and stop based on Peak and Off-Peak business hours, scale out VMs based on number of sessions per CPU core, scale in VMs during Off-Peak hours, leaving the minimum number of session host VMs running. Not available yet for Personal Host Pool type, recommended to have a separate setup for each host pool. ",
+            "text": "Use built-in provided administrative templates for AVD settings configuration",
+            "description": "AVD provides administrative templates for Intune and Active Directory GPO. Using these templates it is possible to centrally several AVD configuration settings: Graphics related data logging, Screen capture protection, RDP Shortpath for managed networks, Watermarking. See companion article in 'More Info' colum for details. NOTE: FSLogix has its own separate template.",
```

Comment:
I think we're missing a verb in  "[...] it is possible to centrally _<something>_ several AVD configuration settings [...]".

Manage or configure?